php-81/Dockerfile

```diff
@@ -95,200 +86,6 @@ RUN set -xe; \
-  | tar xzC ${LIBSSH2_BUILD_DIR} --strip-components=1
-WORKDIR  ${LIBSSH2_BUILD_DIR}/bin/
-RUN CFLAGS="" \
-    CPPFLAGS="-I${INSTALL_DIR}/include -I/usr/include" \
-    LDFLAGS="-L${INSTALL_DIR}/lib64 -L${INSTALL_DIR}/lib" \
-    cmake .. \
-        # Build as a shared library (.so) instead of a static one
-        -DBUILD_SHARED_LIBS=ON \
-        # Build with OpenSSL support
-        -DCRYPTO_BACKEND=OpenSSL \
-        # Build with zlib support
-        -DENABLE_ZLIB_COMPRESSION=ON \
-        -DCMAKE_INSTALL_PREFIX=${INSTALL_DIR} \
-        -DCMAKE_BUILD_TYPE=RELEASE
-RUN cmake  --build . --target install
-
-
-###############################################################################
-# LIBNGHTTP2
-# This adds support for HTTP 2 requests in curl.
-# See https://github.com/brefphp/bref/issues/727 and https://github.com/brefphp/bref/pull/740
-# https://github.com/nghttp2/nghttp2/releases
-# Needs:
-#   - zlib
-#   - OpenSSL
-#   - libxml2
-# Needed by:
-#   - curl
-ENV VERSION_NGHTTP2=1.56.0
-ENV NGHTTP2_BUILD_DIR=${BUILD_DIR}/nghttp2
-RUN set -xe; \
-    mkdir -p ${NGHTTP2_BUILD_DIR}; \
-    curl -Ls https://github.com/nghttp2/nghttp2/releases/download/v${VERSION_NGHTTP2}/nghttp2-${VERSION_NGHTTP2}.tar.gz \
-    | tar xzC ${NGHTTP2_BUILD_DIR} --strip-components=1
-WORKDIR  ${NGHTTP2_BUILD_DIR}/
-RUN CFLAGS="" \
-    CPPFLAGS="-I${INSTALL_DIR}/include -I/usr/include" \
-    LDFLAGS="-L${INSTALL_DIR}/lib64 -L${INSTALL_DIR}/lib" \
-    ./configure \
-    --enable-lib-only \
-    --prefix=${INSTALL_DIR}
-RUN make install
-
-
-###############################################################################
-# CURL
-# # https://github.com/curl/curl/releases
-# # Needs:
-# #   - zlib
-# #   - OpenSSL
-# #   - libssh2
-# #   - libnghttp2
-# # Needed by:
-# #   - php
-ENV VERSION_CURL=8.3.0
-ENV CURL_BUILD_DIR=${BUILD_DIR}/curl
-RUN set -xe; \
-    mkdir -p ${CURL_BUILD_DIR}/bin; \
-    curl -Ls https://github.com/curl/curl/archive/curl-${VERSION_CURL//./_}.tar.gz \
-    | tar xzC ${CURL_BUILD_DIR} --strip-components=1
-WORKDIR  ${CURL_BUILD_DIR}/
-RUN ./buildconf \
- && CFLAGS="" \
-    CPPFLAGS="-I${INSTALL_DIR}/include -I/usr/include" \
-    LDFLAGS="-L${INSTALL_DIR}/lib64 -L${INSTALL_DIR}/lib" \
-    ./configure \
-    --prefix=${INSTALL_DIR} \
-    --with-ca-bundle=${CA_BUNDLE} \
-    --enable-shared \
-    --disable-static \
-    --enable-optimize \
-    --disable-warnings \
-    --disable-dependency-tracking \
-    --with-zlib \
-    --enable-http \
-    --enable-ftp  \
-    --enable-file \
-    --enable-proxy  \
-    --enable-tftp \
-    --enable-ipv6 \
-    --enable-openssl-auto-load-config \
-    --enable-cookies \
-    --with-gnu-ld \
-    --with-ssl \
-    --with-libssh2 \
-    --with-nghttp2
-RUN make install
-
-
-###############################################################################
-# LIBZIP
-# https://github.com/nih-at/libzip/releases
-# Needed by:
-#   - php
-ENV VERSION_ZIP=1.10.1
```

Comment:
Native version: 1.7.3

Reply:
This native version is wayyyyy too old.

Reply:
Could you open an issue up at https://github.com/amazonlinux/amazon-linux-2023/issues detailing the issues with the older version that would require an update?